src/components/common/PillList.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import Pill from './Pill';
+import { Box } from '@mui/material';
+
+export type PillListItem<IdType> = {
+  pillId: IdType;
+  label: string;
+  value: string;
+  onRemovePill?: (pillId: IdType) => void;
+};
+
+type PillListProps<IdType> = {
+  pillData: PillListItem<IdType>[];
+  onRemovePillFromList?: (pillId: IdType) => void;
+};
+
+export default function PillList<IdType>(props: PillListProps<IdType>): JSX.Element {
+  const { pillData, onRemovePillFromList } = props;
+
+  return (
+    <Box display='flex' flexWrap='wrap'>
+      {pillData.map((data) => (
+        <Pill
+          key={String(data.pillId)}
+          id={data.pillId}
```

Comment:
for analytics/readability - we could make this `{`${data.pillId}-${data.label}`}`?

Reply:
Actually, i don't think that would work. The `id` is passed to the `onRemove` handler, so the caller expects it to be the form and type of the `data.pillId` that was passed in.

Reply:
ugh ofc.. i misunderstood this id to be the html id..